app/src/main/res/values/colors.xml

```diff
@@ -3,4 +3,20 @@
     <color name="colorPrimary">#145ca8</color>
     <color name="colorPrimaryDark">#104986</color>
     <color name="colorAccent">#F26419</color>
+
```

Comment:
The code for the edit text does not belong here, please delete everything between the selector tags and the selector tags.